lib/index.js

```diff
@@ -78,18 +75,6 @@ Drip.prototype.track = function(track, fn){
     .end(this.handle(fn));
 };
 
-/**
- * Get campaignId from options or settings.
```

Comment:
Correct. Drip subscribers are not necessarily connected with a campaign, and our current API reflects that.